Example/InstanceID/Tests/FIRInstanceIDTokenOperationsTest.m

```diff
@@ -71,12 +79,24 @@ - (void)setUp {
                                                                     store:_mockStore];
   _instanceID = @"instanceID";
 
+  [self stubInstallations];
```

Comment:
Nit: can you please add a small comment here saying why it's necessary to stub installations?

Reply:
done

Reply:
Thanks!